docs/observers/ecs.md

```diff
@@ -0,0 +1,153 @@
+<!--- GENERATED BY gomplate from scripts/docs/observer-page.md.tmpl --->
+
+# ecs
+
+ Queries the [ECS Task Metadata Endpoint version 2](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v2.html) for running containers.
+
+The Smart Agent needs to run in the same task with the containers to be
+monitored to get access to the data through ECS task metadata endpoint.
+
+## Configuration from Labels
+You can configure monitors by putting special labels on your Docker
```

Comment:
I think we should be more explicit here and say that you **must** put at least one special label to identify a port that is exposed on the container since ECS metadata doesn't contain any port information.